packages/form-js-viewer/src/render/components/form-fields/Taglist.js

```diff
@@ -113,22 +114,44 @@ export default function Taglist(props) {
     }
   };
 
+  const onTagRemoveClick = (event, value) => {
+    const { target } = event;
+
+    deselectValue(value);
+
+    // restore focus if there is no next sibling to focus
+    const nextTag = target.closest('.fjs-taglist-tag').nextSibling;
+    if (!nextTag) {
+      searchbarRef.current.focus();
+    }
+  };
+
   return <div class={ formFieldClasses(type, errors) }>
     <Label
       label={ label }
       id={ prefixId(`${id}-search`, formId) } />
     <div class={ classNames('fjs-taglist', { 'disabled': disabled }) }>
       {!disabled && loadState === LOAD_STATES.LOADED &&
-        values.map((v) => {
-          return (
-            <div class="fjs-taglist-tag" onMouseDown={ (e) => e.preventDefault() }>
-              <span class="fjs-taglist-tag-label">
-                {valueToOptionMap[v] ? valueToOptionMap[v].label : `unexpected value{${v}}`}
-              </span>
-              <span class="fjs-taglist-tag-remove" onMouseDown={ () => deselectValue(v) }><CloseIcon /></span>
-            </div>
-          );
-        })
+        <div class="fjs-taglist-tags">
```

Comment:
I'll have to check if this doesn't break the styling on Tasklist. We have to setup a system for changes on components in order to not break customization

Reply:
I agree with being careful with this kind of change. I introduced this just to ensure the tags are encapsulated from the search input. I can also revert this if you see any problem in Tasklist 👍

Reply:
I just rebased locally with `develop` and tested it on Tasklist. This PR should be good to merge.
I just need to fix the focus state, but it's unrelated to this PR

Reply:
thanks for the confirmation 👍